app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [collection_of_coins, marks,
```

Comment:
You could make a line break after `{` and place the closing bracket `}` on a separate line. Sometimes this way of line-breaking is better for readability.